Update install-vamt.md (#9009)
MicrosoftDocs/windows-itpro-docs: windows/deployment/volume-activation/install-vamt.md

### Install VAMT using the ADK

1. Download the latest version of [Windows 10 ADK](https://docs.microsoft.com/en-us/windows-hardware/get-started/adk-install).

> It is recommended to uninstall ADK and install the latest version, if you use a previous version. Existing data of VAMT is maintained in the respective VAMT database. 

Reviewer: The phrasing appears to be slightly off. I believe a reordering of the sentence clauses could help improve the clarity for English readers.
```suggestion
If an older version is already installed, it is recommended to uninstall the older ADK and install the latest version. Existing VAMT data is maintained in the VAMT database.
```
(I am fairly sure that "respective" is only useful when comparing multiple entities or versions.)
